### PR TITLE
 chore: Add different color for messages from bot

### PR DESCRIPTION
**Pull request by nithindavid**
Users will be able to easily distinguish agent messages with ones sent by bots.

**Preview**
image

### CODE REVIEW

File: app/javascript/dashboard/components/widgets/conversation/Message.vue [hunk 1]
```diff
@@ -197,11 +197,15 @@ export default {
...
         'is-image': this.hasImageAttachment,
         'is-text': this.hasText,
+        'is-from-bot': this.isSentByBot,
       };
     },
     isPending() {
       return this.data.status === MESSAGE_STATUS.PROGRESS;
     },
+    isSentByBot() {
+      return !this.sender.type || this.sender.type === 'bot';
```
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/conversation/Message.vue:207]: sender can be null

**Review by pranavrajs — APPROVED** [review]

**Review by sojan-official — APPROVED** [review]

**Review by muhsin-k — APPROVED** [review]

File: app/javascript/dashboard/components/widgets/conversation/Message.vue (hunk 1 above)
Comment (pranavrajs) [reply app/javascript/dashboard/components/widgets/conversation/Message.vue:207]: @nithindavid Sorry about this, this was not needed. This was handled already in the computed property.

**Review by pranavrajs — CHANGES_REQUESTED** [review]
On QA, I found that the message turns violet first and then becomes blue. Please see the attached video.


https://user-images.githubusercontent.com/2246121/120775981-5e441b80-c541-11eb-8e16-33bd6e6343b9.mp4

**Comment by nithindavid** [comment]
@pranavrajs Thank for pointing out! I have added the fix.

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**